scripts/sign.ps1

```diff
@@ -16,13 +16,6 @@ Foreach-Object {
 }
 Write-Host "[Completed]Signing .NET 4.6 assemblies"
 
-Write-Host "Signing .NET Core 2 assemblies"
-Get-ChildItem "build\sonarscanner-msbuild-netcoreapp2.0\Sonar*.dll" | 
-Foreach-Object {
-    & $env:SIGNTOOL_PATH sign /fd SHA256 /f $env:PFX_PATH /p $env:PFX_PASSWORD /tr http://timestamp.digicert.com?alg=sha256 $_.FullName
-}
-Write-Host "[Completed] Signing .NET Core 2 assemblies"
-
 Write-Host "Signing .NET Core 3 assemblies"
 Get-ChildItem "build\sonarscanner-msbuild-netcoreapp3.0\Sonar*.dll" | 
```

Comment:
Educational: given the upcoming change of folder, that won't contain the target in the name anymore, this target-specific code is going to be changed into a single, target-agnostic code, correct?

Reply:
yes, exactly, we have https://github.com/SonarSource/sonar-scanner-msbuild/issues/1770 to handle that